Source/Fuse.Controls.Native/Android/SystemScreenConfigAndroidImpl.uno

```diff
@@ -0,0 +1,104 @@
+using Fuse.Controls.Native;
+
+namespace Fuse.Controls.Native
+{
+	extern(Android) internal class SystemScreenConfigAndroidImpl
+	{
+		private static SystemUiVisibility.Flag _wantedFlags = SystemUiVisibility.Flag.None;
+		private static SystemScreenConfig config;
+
+		public static void Init(SystemScreenConfig config)
+		{
+			SystemScreenConfigAndroidImpl.config = config;
+			SystemUiVisibility.VisibilityChanged += VisibilityChanged;
+		}
+
+		public static void DeInit()
+		{
+			SystemUiVisibility.VisibilityChanged -= VisibilityChanged;
+			SystemScreenConfigAndroidImpl.config = null;
+		}
+
+		public static void TimerDone()
+		{
+			SystemUiVisibility.Flags = _wantedFlags;
+		}
+
+		public static void SetShowState(SystemScreenConfig.Visibility state)
+		{
+			switch(state)
+			{
+				case SystemScreenConfig.Visibility.None:
+					SystemUiVisibility.Flags = SystemUiVisibility.Flag.Fullscreen | SystemUiVisibility.Flag.HideNavigation;
+					_wantedFlags = SystemUiVisibility.Flag.Fullscreen | SystemUiVisibility.Flag.HideNavigation;
+				break;
+				case SystemScreenConfig.Visibility.Minimal:
+					SystemUiVisibility.Flags = SystemUiVisibility.Flag.HideNavigation;
+					_wantedFlags = SystemUiVisibility.Flag.HideNavigation;
+				break;
+				case SystemScreenConfig.Visibility.Full:
+					SystemUiVisibility.Flags = SystemUiVisibility.Flag.None;
+					_wantedFlags = SystemUiVisibility.Flag.None;
+				break;
+			}
+		}
+
+		public static void SetStatusState(bool val)
+		{
+			if(val)
+			{
+				_wantedFlags = _wantedFlags ^ SystemUiVisibility.Flag.Fullscreen;
+				SystemUiVisibility.Flags = SystemUiVisibility.Flags ^ SystemUiVisibility.Flag.Fullscreen;
```

Comment:
@petterroea: Sure, but this code is run when a property is changed, and I believe that should yield an immediate update to the wanted flags, no? In other words, I think the timer should only do it's work unless some other update happened in the meantime...